docs/IR.md

```diff
@@ -89,6 +91,15 @@ Each model has the following components:
 
  Models MUST specify a domain and use reverse domain names based on the responsible organization's identity, the same convention that is traditionally used for naming Java packages.
  
+__Note: Exploring an ONNX file__
```

Comment:
On this topic, may be it would be useful to point to the graphic visualizer (Netron) as well: https://github.com/lutzroeder/netron